mechanics/equipment/weapons/README.md

```diff
@@ -4,6 +4,16 @@
 
 Weapons can be refined by combining a weapon with one of the same type. This increases the potency of the effect of the weapon by approximately 25%. The final refinement level is equal to the sum of the refinement level of the material and the refinement level of the base weapon. The level of the base weapon is the only one maintained. The base weapon determines the cost of refinements, so it is recommended to use lower refinement weapons as the base unless trying to preserve levels.
 
+## Weapon Effect Trigger Chance Against Multiple Enemies
+
+### Sacrificial Series
+
+The Sacrificial Weapons' effect, **Composed**, has its trigger chance calculated separately when hitting multiple targets, thus increasing its overall chance against multiple enemies.
+
+###  Favonius Series
+
+The Favonius Weapons' effect, **Windfall**, has its trigger chance calculated separately when hitting multiple targets, thus increasing its overall chance against multiple enemies.
+
```

Comment:
Doesn't seem to be follow what's generally in this page which is GENERAL weapon mechanics. Probably just leave this in evidence.

Reply:
Sure, it's not a General Weapon Mechanic. But IT IS a _general mechanic_ that applies to all types of weapons within the weapon series.

As more and more weapons under the same series also share similar mechanics (like the millennial movement series), we need this page to address sub-general mechanics like these as well.

Reply:
So far for mechanics that affect weapon series, we just put it in the tabs of all relevant weapons. But if it's desired, we can set up a weapon series page that lists: weapon series, what weapons are part of what series, and the mechanics of that weapon series. This would also allow splitting up the honestly overly long weapon evidence vault we currently have.

Reply:
aight will look at it to find the optimal route